litmus-portal/frontend/package.json

```diff
@@ -21,9 +21,6 @@
     "brace": "^0.11.1",
     "date-fns": "^2.14.0",
     "graphql": "^15.3.0",
-    "i18next": "^19.7.0",
```

Comment:
why was i18 removed @ishangupta-ds 
@chandankumar4 was this something expected?

Reply:
We need this!

Reply:
I think @ishangupta-ds did not pull the new changes or rejected the incoming changes.

Reply:
No i did pull the changes, an merged my code, as all translations were just delete and there were no conflicts so this happened, had to reset to @chandankumar4  's commit. Will raise a new PR shortly with the same changes and translation files included. Incorporating the suggested changes to this PR @chandankumar4 @arkajyotiMukherjee @gdsoumya @oumkale cc: @rajdas98